### ISSUE TITLE
Rethink chirp mass range in `pycbc_optimize_snr`?

### ISSUE THREAD

**Issue by titodalcanton** [gwastro/pycbc#3928]
@spxiwh notes:

```python
minchirp = mchirp * (1 - mchirp / 50.0)
maxchirp = mchirp * (1 + mchirp / 50.0)
```

> It looks like there's a bug here. I think this was supposed to be fractional bounds on mchirp (maybe ones that increase with the mchirp??). But as a specific fail case: if the mchirp is 50, this will search down to a mchirp of 0 and then things will break!
> Although I guess the lines below catch that, but I still think the bounds aren't ideal!

**Comment by titodalcanton** [comment]
I did in fact see this error while playing with `pycbc_optimize_snr`, though I am not sure if this is a consequence of what @spxiwh said.
```
Traceback (most recent call last):
  File "/home/tito/virtualenvs/pycbc/bin/pycbc_optimize_snr", line 350, in <module>
    opt_params = optimize_func(bounds, args, extra_args)
  File "/home/tito/virtualenvs/pycbc/bin/pycbc_optimize_snr", line 135, in optimize_di
    results = differential_evolution(
  File "/home/tito/virtualenvs/pycbc/lib/python3.9/site-packages/scipy/optimize/_differentialevolution.py", line 329, in differential_evolution
    ret = solver.solve()
  File "/home/tito/virtualenvs/pycbc/lib/python3.9/site-packages/scipy/optimize/_differentialevolution.py", line 903, in solve
    result = minimize(self.func,
  File "/home/tito/virtualenvs/pycbc/lib/python3.9/site-packages/scipy/optimize/_minimize.py", line 623, in minimize
    return _minimize_lbfgsb(fun, x0, args, jac, bounds,
  File "/home/tito/virtualenvs/pycbc/lib/python3.9/site-packages/scipy/optimize/lbfgsb.py", line 294, in _minimize_lbfgsb
    raise ValueError("LBFGSB - one of the lower bounds is greater than an upper bound.")
ValueError: LBFGSB - one of the lower bounds is greater than an upper bound.
```

**Comment by tdent** [comment]
Well, maybe an idea behind the formula was that the highest chirp mass in the live bank was less than 50.

**Comment by tdent** [comment]
We could try and fix this mathematically by noting that 1 \pm mchirp/50 is the first order expansion of exp(\pm mchirp/50), and if you use the exponential instead of the linear function the minimum never gets to zero.  Though for mchirp >> 50 it does get very close to 0.

**Comment by tdent** [comment]
Seems like Veronica's injection results in the most recent version of https://dcc.ligo.org/LIGO-P2200060 could be used to inform this.

**Comment by tdent** [comment]
Closed by #4338. No-one spotted the terrible mchirp<1 corner case for a long time ...
